operator/v1/0000_01_operator.open-cluster-management.io_clustermanagers.crd.yaml

```diff
@@ -356,6 +356,19 @@ spec:
                       - feature
                       type: object
                     type: array
+                  workDriver:
+                    default: kube
+                    description: WorkDriver represents the type of work driver. Possible
+                      values are kube, mqtt, or grpc. If not provided, the default
+                      value is kube.
+                    type: string
+                  workDriverConfig:
+                    description: 'WorkDriverConfig is the secret containing the configuration
+                      file for the work driver in YAML format. It must not be empty
+                      if the driver is not kube. For detailed configuration content,
+                      refer to the specific work driver options package: MQTT: https://github.com/open-cluster-management-io/sdk-go/blob/00a94671ced1c17d2ca2b5fad2f4baab282a7d3c/pkg/cloudevents/generic/options/mqtt/options.go#L46-L76
+                      GRPC: https://github.com/open-cluster-management-io/sdk-go/blob/00a94671ced1c17d2ca2b5fad2f4baab282a7d3c/pkg/cloudevents/generic/options/grpc/options.go#L30-L40'
```

Comment:
It is better to show a simple example for MQTT and GRPC here.

Reply:
added examples.